cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -449,7 +449,11 @@ func searchFilesInRepos(ctx context.Context, args *search.TextParameters, stream
 				repos = append(repos, repo)
 			}
 
-			return callSearcherOverRepos(ctx, args, stream, repos, nil)
+			return callSearcherOverRepos(ctx, args, stream, repos, nil, true)
+		})
+
+		g.Go(func() error {
+			return callSearcherOverRepos(ctx, args, stream, indexed.Unindexed, nil, false)
```

Comment:
A bit of a gotcha, but searcherRepos is the list of repos you are allowed to search unindexed. I'll actually do a little PR which fixes this issue. It would read much nicer if you could just directly use indexed.Unindexed.

```suggestion
			return callSearcherOverRepos(ctx, args, stream, searcherRepos, nil, false)
```

Reply:
Unified in https://github.com/sourcegraph/sourcegraph/pull/17980 so ignore my suggestion since once that lands `indexed.Unindexed` will be correct.